release-notes/2024-04/Drop-workorder-database.md

```diff
@@ -0,0 +1,28 @@
+# Dropping the work order database
+
+## Assumptions
+
+- The workorder service is enabled but not in use, no data needs to be retained.
+- SystemLink Enterprise deployment uses in-cluster, per-service mongo databases.
+- Administrator has `kubectl` access to the Kubernetes cluster from a local bash shell.
+- Administrator has permission to stop and start deployments and to exec into pods.
+
+## Procedure
+
+1. Complete the upgrade to the 2024-04 helm chart
+1. Stop the work order deployment
+   `kubectl scale deployment <release-name>-workorder` --replicas=0
+1. Wait for all `<release-name>-workorder` pods to stop.
+1. Determine which mongo pod is the primary replicaset.
```

Comment:
```suggestion
1. Determine which MongoDB pod is the primary replicaset.
```

Also, is each replica its own pod or are they collectively a pod?

Reply:
I meant "replica" here, not "replicaset". Each replica is a pod.